crates/sparrow-session/src/lib.rs

```diff
@@ -9,3 +9,5 @@ pub use execution::Execution;
 pub use expr::{Expr, Literal};
 pub use session::{ExecutionOptions, Session};
 pub use table::Table;
+
+// pub type Udf = dyn sparrow_instructions::Udf;
```

Comment:
The type alias was not allowed when implementing the trait. Perhaps was doing it incorrectly, though

Reply:
Ah, yeah -- the reason is that it changes `impl Udf (the trait) for ...` into `imp dyn Udf for ...`... but you can only implement traits.